1_2.js

```diff
@@ -0,0 +1,10 @@
+function solution(num) {
+    if (num%2){
+        var answer = "Odd";
```

Comment:
```suggestion
        return "Odd";
```
굳이 변수에 안담고 바로 리턴해줘도 될 것 같습니다.